### PR TITLE
Fixes lavaland syndicate station's explosives being able to be picked up

### PR DESCRIPTION
**Pull request by XeonMations**
## About The Pull Request

Makes the actual explosives a subtype, as well as making it so they cant be picked up.

## Why It's Good For The Game

no free explosives for you, lavaland

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

https://github.com/user-attachments/assets/271b84c4-99f2-42d8-b66a-886a1d4f6833

https://github.com/user-attachments/assets/db75f466-2aba-4d04-847f-116186391569

</details>

## Changelog
:cl: XeonMations
fix: The syndicate lavaland base's explosives are no longer able to be picked up.
/:cl:

### CODE REVIEW

**Comment by Fronsis** [comment]
Hm, aren't a few payloads missing? iirc the whole base is supposed to be engulfed entirely in flames rather than just half of the base (based on the last second of your view range on the 2nd video)

**Comment by XeonMations** [comment]
> Hm, aren't a few payloads missing? iirc the whole base is supposed to be engulfed entirely in flames rather than just half of the base (based on the last second of your view range on the 2nd video)

I dunno. I did not change  the mapping at all. I just made them invisibile to the player ingame.

File: code/game/turfs/closed/wall/mineral_walls.dm [hunk 1]
```diff
@@ -352,17 +352,23 @@
...
 /turf/closed/wall/mineral/plastitanium/explosive
-	var/obj/item/bombcore/large/bombcore
+	var/obj/item/bombcore/large/syndicate_base/bombcore
 
 /turf/closed/wall/mineral/plastitanium/explosive/Initialize(mapload)
 	. = ..()
-	bombcore = new(get_turf(src))
-	bombcore.installed = TRUE
+	bombcore = new /obj/item/bombcore/large/syndicate_base(get_turf(src))
+	bombcore.forceMove(src)
```
Comment (PowerfulBacon) [inline code/game/turfs/closed/wall/mineral_walls.dm]:
```suggestion
	bombcore = new /obj/item/bombcore/large/syndicate_base(src)
```
Is there any reason not to directly initialise it inside of src?

File: code/game/machinery/syndicatebomb.dm [hunk 2]
```diff
@@ -399,6 +399,12 @@
 	range_light = 20
 	range_flame = 20
 
+// Special bomb core for inside the lavaland syndicate base walls. Players should not be able to interact with this.
+/obj/item/bombcore/large/syndicate_base
+	installed = TRUE
+	anchored = TRUE
+	invisibility = INVISIBILITY_OBSERVER
```
Comment (PowerfulBacon) [inline code/game/machinery/syndicatebomb.dm:406]: Is this actually needed if the bombcore is force moved inside the wall anyway?

Comment (XeonMations) [reply code/game/machinery/syndicatebomb.dm:406]: Yes, because it does still appear in the context menu.

**Comment by XeonMations** [comment]
image

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**